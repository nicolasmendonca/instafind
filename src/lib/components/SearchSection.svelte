<section
	class="py-16 flex flex-col space-y-12 items-center justify-between relative bg-search-section-bg bg-center bg-cover"
>
	<h1 class="text-white font-cursive text-3xl">Encontr√° lo que necesitas</h1>

	<form
		class="bg-white flex items-center rounded-lg focus-within:ring focus-within:border-purple-300 "
	>
		<img src="/static/icons/lens.svg" alt="" class="w-4 h-4 ml-2" />
		<input
			type="search"
			class="px-4 rounded-md focus:outline-none"
			placeholder="Ropa para casamiento"
		/>
		<button
			type="submit"
			class="right-0 flex-grow text-xs bg-purple-600 text-white py-3 px-2 rounded-r-lg"
			>Encontralo!</button
		>
	</form>
</section>
