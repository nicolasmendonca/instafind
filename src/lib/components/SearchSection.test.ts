import SearchSection from './SearchSection.svelte'
import {render} from '@testing-library/svelte'

test('matches the snapshot', () => {
  const {container} = render(SearchSection)
  expect(container).toMatchSnapshot()
})
