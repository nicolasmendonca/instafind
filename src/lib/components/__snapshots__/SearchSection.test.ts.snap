// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<body>
  <div>
    <section
      class="py-16 flex flex-col space-y-12 items-center justify-between relative bg-search-section-bg bg-center bg-cover"
    >
      <h1
        class="text-white font-cursive text-3xl"
      >
        Encontr√° lo que necesitas
      </h1>
       
      <form
        class="bg-white flex items-center rounded-lg focus-within:ring focus-within:border-purple-300 "
      >
        <img
          alt=""
          class="w-4 h-4 ml-2"
          src="/static/icons/lens.svg"
        />
         
        <input
          class="px-4 rounded-md focus:outline-none"
          placeholder="Ropa para casamiento"
          type="search"
        />
         
        <button
          class="right-0 flex-grow text-xs bg-purple-600 text-white py-3 px-2 rounded-r-lg"
          type="submit"
        >
          Encontralo!
        </button>
      </form>
    </section>
  </div>
</body>
`;
